test/controllers/demo_user_controller_test.rb

```diff
@@ -407,6 +407,53 @@ class DemoUserControllerTest < ActionDispatch::IntegrationTest
           DeviseTokenAuth.headers_names[:'access-token'] = 'access-token'
         end
       end
+
+      describe 'maximum concurrent devices per user' do
+        before do
+          # Set the max_number_of_devices to a lower number
+          #  to expedite tests! (Default is 10)
+          DeviseTokenAuth.max_number_of_devices = 5
+
+          # @max_devices = DeviseTokenAuth.max_number_of_devices
```

Comment:
remove the commented line, or is this documentation?

Reply:
It is cruft and I didn't mean to leave it in the commit! 😆